### ISSUE TITLE
Multiple Blockchain Options

### ISSUE THREAD

**Issue by brown7477** [davehagler/nftlogin#1]
Bare with me with this as I am not sure how to comment or ask questions on GitHub.
Anyways, My question is it possible to use this with other blockchains other than ETH?
I am working on some Solana or Possibly POLYGON projects and would love to have this for login/signup.

**Comment by davehagler** [comment]
Yes, I’m planning to add more chains. Polygon for sure and I’m working on it now hoping to have it this working this week. I’d like to add Solana but it’s going to be more difficult so will take a little longer

**Comment by brown7477** [comment]
Very Cool.
I am trying to test it now with MATIC, but for some reason it is not wanting to connect to my MM wallet.

**Comment by brown7477** [comment]
Also a testnet option for testing would be incredible.

**Comment by brown7477** [comment]
Hey Dave,
How are things coming along?
Any luck with being able to use this with Testnet NFTs for testing purposes only?
I am really looking for a clean option for membership registrations.
